driver/apdu/pcsc.c

```diff
@@ -29,6 +29,14 @@ static SCARDCONTEXT pcsc_ctx;
 static SCARDHANDLE pcsc_hCard;
 static LPSTR pcsc_mszReaders;
 
+static void pcsc_error_text(const char *method, const int32_t code) {
```

Comment:
```suggestion
static void pcsc_error(const char *method, const int32_t code) {
```
`_text` is confusing.